### PR TITLE
browserstack-bitrise-xcui-step 1.0.1

### PR DESCRIPTION
**Pull request by deepwired**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/browserstack-bitrise-xcui-step/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    title: BrowserStack accesskey
+- app_ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: Path of the app (.ipa) file
+    is_expand: true
+    is_required: true
+    summary: Path to the app (.ipa) file
+    title: iOS app under test
+- opts:
+    description: Path of the output bundle (.app) file. Add either the $BITRISE_TEST_BUNDLE_PATH
```
Comment (godrei) [inline steps/browserstack-bitrise-xcui-step/1.0.1/step.yml:57]: NOTE: descriptions are presented as a markdown content, `_` char should be escaped in `$BITRISE_TEST_BUNDLE_PATH`: -> `$BITRISE\_TEST\_BUNDLE\_PATH` or it should be formatted as a code: 
```
`$BITRISE_TEST_BUNDLE_PATH`
```

**Comment by godrei** [comment]
Hi @deepwired  ,
thanks for the new step version! 
I found a small issue in the step.yml, that could be fixed sometime in the future.

**Comment by godrei** [comment]
The PR looks good to me, I would like to ask you to sign the Contributor License Agreement (see PR checks), after that I can merge this PR.

**Comment by deepwired** [comment]
@godrei done

**Comment by deepwired** [comment]
Would it be possible to approve/merge and go live today?

**Comment by deepwired** [comment]
gentle reminder ^^

**Review by tothszabi — APPROVED** [review]

**Pull request merged**